block.py

```diff
@@ -1,34 +1,60 @@
+#!/usr/bin/env python3
+
+import argparse
+import glob
 import os
-import sys
+from pathlib import Path
 import subprocess
+import sys
 from zipfile import ZipFile
-from pathlib import Path
-import argparse
 
-parser = argparse.ArgumentParser(description='Tool for garbling PII in for PPRL purposes in the CODI project')
-parser.add_argument('--schema', nargs=1, required=True, help='Path to blocking schema')
-args = parser.parse_args()
 
-if not os.path.exists('blocking-output'):
-  os.mkdir('blocking-output')
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description="Tool for garbling PII for PPRL purposes in the CODI project"
+    )
+    parser.add_argument(
+        "--schemafile", default="example-schema/blocking-schema/lambda.json",
+        help="Path to blocking schema. Default: example-schema/blocking-schema/lambda.json"
+    )
+    parser.add_argument(
+        '--clkpath', default="output/",
+         help="Specify a folder containing clks. Default is output/"
+    )
+    args = parser.parse_args()
+    if not Path(args.schemafile).exists():
+        parser.error("Unable to find schema file: " + args.schemafile)
+    return args
+
+
+def block_individuals(args):
+    os.makedirs('temp-data', exist_ok=True)
+    os.makedirs('output', exist_ok=True)
+    schema_file = Path(args.schemafile)
+    clk_files = glob.glob(args.clkpath + "/*.json")
```

Comment:
The default `args.clkpath` includes a trailing slash. probably not an error here but should probably just be one or the other - or neither, instead maybe using pathlib or `os.path.join()` or `os.sep`.